lib/chef/provisioning/aws_driver/aws_taggable.rb

```diff
@@ -0,0 +1,18 @@
+module Chef::Provisioning::AWSDriver
+# This module is meant to be included in a resource that is taggable
+# This will add the appropriate attribute that can be converged by the provider
+# TODO it would be nice to not have two seperate modules (taggable/tagger)
```

Comment:
:+1: Thanks!  I will keep the resource and provider modifications separate for now